doc/user/layouts/shortcodes/network-security/privatelink-kafka.md

```diff
@@ -151,4 +151,7 @@ CREATE CONNECTION kafka_connection TO KAFKA (
 );
 ```
 
-The `(PORT <port_number>)` value must match the port that you used when creating the **TCP listener** in the Network Load Balancer. Be sure to specify the correct availability zone for each broker.
+There are a few important things to note:
```

Comment:
There's a few more places where this needs to be adapted. I can push the changes in tomorrow, since those are still fresh in my head from the cross-region docs PR!